resources/about_storage.md

```diff
@@ -0,0 +1,57 @@
+# About storage
+
+::: tip TLDR
+
+- If the database does not shrink after deleting documents or indexes, it is expected behavior. You are not losing space, MeiliSearch is keeping this space for performance reasons
+- You should have the same amount of RAM than the space taken on disk by MeiliSearch for optimal performances.
+  :::
+
+MeiliSearch is a database. It stores the indexed documents along with the data needed to perform lightning search.
+
+Writing a database is hard, and we do not want to reinvent the wheel, so MeiliSearch uses a storage engine under the hood. Using a storage engine allows MeiliSearch to focus on improving search relevancy and search performance while abstracting the complicated task of creating, reading and updating documents on disk, and in memory.
+
+## LMDB
+
+The storage engine of MeiliSearch is [LMDB](http://www.lmdb.tech/doc/). LMDB is a transactional key-value store written in C that was developed for OpenLDAP, and it has ACID properties.
+We choose LMDB after successfully (or not) tried MeiliSearch with [Sled](https://github.com/spacejam/sled) and [RocksDB](https://rocksdb.org/) and decided to move on with LMDB because it is the best combination of performance and stability for Meilisearch.
+
+### Memory mapping
+
+LMDB stores its data in a [memory-mapped file](https://en.wikipedia.org/wiki/Memory-mapped_file). All data fetched from LMDB are returned straight from the memory map, which means there is no memory allocation or memory copy during data fetches.
```

Comment:
All data fetched from LMDB `is` returned

Reply:
💡 https://www.theguardian.com/news/datablog/2010/jul/16/data-plural-singular